src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
yep, I think the name of the method can change - I might call DynamoViewModel.SaveWorkspace(workspaces[1]) or something like that as a user of the API.

If you want it to only save the current workspace then name it that.